integration/raft/client.go

```diff
@@ -25,7 +25,7 @@ func FetchBlock(n *nwo.Network, o *nwo.Orderer, seq uint64, channel string) *com
 		var err error
 		blk, err = ordererclient.Deliver(n, o, denv)
```

Comment:
@guoger The creation of the orderer client inside of Deliver is throttled by the network.

Deliver: https://github.com/hyperledger/fabric/blob/76763d1b8900df79080765453f944122dd7d68aa/integration/ordererclient/orderer_client.go#L20
OrdererClientConn: https://github.com/hyperledger/fabric/blob/76763d1b8900df79080765453f944122dd7d68aa/integration/nwo/network.go#L1486-L1501